src/ui/Templates/TemplateFromAScriptTag.ts

```diff
@@ -65,21 +65,23 @@ export class TemplateFromAScriptTag {
     this.template.addFields(neededFieldsForComponents);
   }
 
-  toHtmlElement(): HTMLElement {
-    var script = $$('code');
-    let condition = $$(this.scriptTag).getAttribute('data-condition');
+  toHtmlElement(container?: Dom): HTMLElement {
+    if (!container) {
+      container = $$('code');
+    }
+    const condition = $$(this.scriptTag).getAttribute('data-condition');
     if (condition) {
-      script.setAttribute('data-condition', condition);
+      container.setAttribute('data-condition', condition);
     }
-    script.setHtml(this.scriptTag.innerHTML);
-    return script.el;
+    container.setHtml(this.scriptTag.innerHTML);
+    return container.el;
   }
 
   parseFieldsAttributes(): IFieldsToMatch[] {
-    let dataSet = this.scriptTag.dataset;
+    const dataSet = this.scriptTag.dataset;
     return _.chain(dataSet)
       .map((value, key: string) => {
-        let match = key.match(/field([a-z0-9]*)/i);
+        const match = key.match(/field([a-z0-9]*)/i);
         if (match) {
           let values;
```

Comment:
Could values be const instead of let? I couldn't find code that modifies it's value.